src/main/kotlin/net/irisshaders/lilybot/extensions/util/ThreadControl.kt

```diff
@@ -151,8 +142,27 @@ class ThreadControl : Extension() {
 	inner class ThreadArchiveArgs : Arguments() {
 		val lock by defaultingBoolean {
 			name = "lock"
-			description = "Whether to lock this thread, if you are a moderator. Default is false"
+			description = "Whether to lock the thread if you are a moderator. Default is false"
 			defaultValue = false
 		}
 	}
+
+	inner class ThreadTransferArgs : Arguments() {
+		val newOwner by member {
+			name = "newOwner"
+			description = "The user you want to transfer ownership of the thread to"
+		}
+	}
+
+	private suspend fun EphemeralSlashCommandContext<*>.ownsThreadOrModerator
+				(inputThread: ThreadChannel, inputMember: Member): Boolean {
```

Comment:
```suggestion
	private suspend fun EphemeralSlashCommandContext<*>.ownsThreadOrModerator(
			inputThread: ThreadChannel, 
			inputMember: Member
	): Boolean {
```
Just to fix the formatting to match how all other functions are formatted